src/utils/quickPickFile.ts

```diff
@@ -51,14 +52,27 @@ export async function resolveFileItem(rootFolder: vscode.WorkspaceFolder, fileUr
     } else {
         let items: Item[] = uris.map(uri => createFileItem(rootFolder, uri));
         if (items.length === 1) {
-            return items[0];
+            return items;
         } else {
-            const res: vscode.QuickPickItem = await ext.ui.showQuickPick(items, { placeHolder: message });
-            return <Item>res;
+            return await ext.ui.showQuickPick<Item>(items, { placeHolder: message, canPickMany: true });
         }
     }
 }
 
+export async function resolveFileItems(rootFolder: vscode.WorkspaceFolder, fileUri: vscode.Uri | undefined, globPatterns: string[], message: string): Promise<Item[] | undefined> {
```

Comment:
I don't think this actually needs to be exported, since there's no external callers.